src/screens/signUp/helper/toast.js

```diff
@@ -0,0 +1,14 @@
+import { toast } from "react-toastify";
```

Comment:
helper folder should not be inside the module folder make it global. for better reuse ability